diff --git a/target/product/aosp_base_telephony.mk b/target/product/aosp_base_telephony.mk
index 8e5cc4cbdf..f1ddd7718c 100644
--- a/target/product/aosp_base_telephony.mk
+++ b/target/product/aosp_base_telephony.mk
@@ -16,5 +16,4 @@
 $(call inherit-product, $(SRC_TARGET_DIR)/product/full_base_telephony.mk)
 
 PRODUCT_PACKAGES += \
-    Messaging \
     Stk
diff --git a/target/product/aosp_product.mk b/target/product/aosp_product.mk
index c54d58ddf6..fe09fa2969 100644
--- a/target/product/aosp_product.mk
+++ b/target/product/aosp_product.mk
@@ -31,7 +31,6 @@ PRODUCT_PRODUCT_PROPERTIES += \
 # More AOSP packages
 PRODUCT_PACKAGES += \
     initial-package-stopped-states-aosp.xml \
-    Messaging \
     PhotoTable \
     preinstalled-packages-platform-aosp-product.xml \
     ThemePicker \
diff --git a/target/product/handheld_product.mk b/target/product/handheld_product.mk
index 2a4602d43b..bd9a088fea 100644
--- a/target/product/handheld_product.mk
+++ b/target/product/handheld_product.mk
@@ -22,21 +22,25 @@ $(call inherit-product, $(SRC_TARGET_DIR)/product/media_product.mk)
 
 # /product packages
 PRODUCT_PACKAGES += \
-    AppStore \
-    Auditor \
-    Calendar \
-    Camera \
-    Contacts \
-    DeskClock \
-    ExactCalculator \
-    Gallery2 \
-    InfoApp \
-    LatinIME \
-    Music \
-    PdfViewerGOS \
+    ModernAppsWeb \
+    ModernAppsCamera \
+    ModernAppsPdf \
+    ModernAppsContacts \
+    ModernAppsCalculator \
+    ModernAppsClock \
+    ModernAppsFiles \
+    ModernAppsPhotos \
+    ModernAppsStore \
+    ModernAppsKeyboard \
+    ModernAppsSpeech \
+    ModernAppsCalendar \
+    ModernAppsMusic \
+    ModernAppsCommunicate \
+    ModernAppsEuicc \
+    ModernAppsBackup \
+    ModernAppsNetworkLocation \
     preinstalled-packages-platform-handheld-product.xml \
     SettingsIntelligence \
-    SpeechServices \
     talkback \
     ThemePicker \
     ThemesStub \
diff --git a/target/product/handheld_system.mk b/target/product/handheld_system.mk
index 029bdd6ef3..b9729321b2 100644
--- a/target/product/handheld_system.mk
+++ b/target/product/handheld_system.mk
@@ -48,7 +48,6 @@ PRODUCT_PACKAGES += \
     CredentialManager \
     DeviceAsWebcam \
     DeviceDiagnostics \
-    DocumentsUI \
     DownloadProviderUi \
     EasterEgg \
     ExternalStorageProvider \
@@ -62,7 +61,6 @@ PRODUCT_PACKAGES += \
     MmsService \
     MtpService \
     MusicFX \
-    NetworkLocation \
     PacProcessor \
     preinstalled-packages-platform-handheld-system.xml \
     PrintRecommendationService \
diff --git a/target/product/media_system.mk b/target/product/media_system.mk
index 424844a429..8cd1714de2 100644
--- a/target/product/media_system.mk
+++ b/target/product/media_system.mk
@@ -40,7 +40,6 @@ ifeq ($(OFFICIAL_BUILD),true)
     PRODUCT_PACKAGES += Updater
 endif
 
-PRODUCT_PACKAGES += Seedvault
 
 # Allowlist for system packages included in media_system.mk
 PRODUCT_PACKAGES += preinstalled-packages-media-system.xml
diff --git a/target/product/telephony_product.mk b/target/product/telephony_product.mk
index aa70f46797..0705511932 100644
--- a/target/product/telephony_product.mk
+++ b/target/product/telephony_product.mk
@@ -19,6 +19,5 @@
 
 # /product packages
 PRODUCT_PACKAGES += \
-    Dialer \
     ImsServiceEntitlement \
     preinstalled-packages-platform-telephony-product.xml
